docs/sphinx/manual/building_hpx.rst

```diff
@@ -279,186 +109,19 @@ listed below.
    localities. Please set the CMake variable ``MPI_CXX_COMPILER`` to your MPI
    C++ compiler wrapper if not detected automatically.
 
-.. list-table:: Highly recommended optional software prerequisites for |hpx| on
-   Linux systems
+.. list-table:: Optional software prerequisites for |hpx|
 
    * * Name
      * Minimum version
-     * Notes
    * * |google_perftools|_
      * 1.7.1
-     * Used as a replacement for the system allocator, and for allocation
-       diagnostics.
-   * * |libunwind|_
-     * 0.97
-     * Dependency of google-perftools on x86-64, used for stack unwinding.
-   * * |openmpi|_
-     * 1.8.0
-     * Can be used as a highspeed communication library backend for the
-       parcelport.
-
-.. note::
-
-   When using OpenMPI please note that Ubuntu (notably 18.04 LTS) and older
-   Debian ship an OpenMPI 2.x built with ``--enable-heterogeneous`` which may
-   cause communication failures at runtime and should not be used.
-
-.. list-table:: Optional software prerequisites for |hpx| on Linux systems
-
-   * * Name
-     * Minimum version
-     * Notes
    * * |papi|
      *
```

Comment:
Could you move PAPI below gperftools, jemalloc, and mimalloc, please? gperftools, jemalloc, and mimalloc are all allocators so they can be grouped together.